docs/upgrade.html

```diff
@@ -65,6 +65,8 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
                         </li>
                         <li>The function <code>onNewBatch</code> in <code>org.apache.kafka.clients.producer.Partitioner</code> class was removed.
                         </li>
+                        <li>The <code>control.plane.listener.name</code> config was removed.
```

Comment:
@chia7712 My point is that a lot more than this config will be removed as part of ZK removal. Why are we listing this one specifically? As I stated elsewhere, we should have a note saying that ZK support has been removed with a link to all the implications as well as the migration details.

Reply:
> My point is that a lot more than this config will be removed as part of ZK removal. Why are we listing this one specifically? As I stated elsewhere, we should have a note saying that ZK support has been removed with a link to all the implications as well as the migration details.

I completely agree that comprehensive migration details are necessary. However, since the documentation for these details may require further discussion, let's address them as a follow-up to avoid delaying the code cleanup. @TaiJuWu has already opened https://issues.apache.org/jira/browse/KAFKA-18364 to track the documentation effort.

In the meantime, could we create a temporary section listing the configurations removed due to Zookeeper? Once the migration details are finalized in KAFKA-18364, this temporary section can be replaced with a more thorough explanation. The temporary section is a note in KAFKA-18364 to ensure all implications are considered.

Reply:
Yes, I think we can make a note in that JIRA instead of the upgrade notes.

Reply:
Remove upgrade and JIRA already link to [KAFKA-18364](https://issues.apache.org/jira/browse/KAFKA-18364)

Reply:
@frankvicky could you please file a minor to remove the password configs form the upgrade.html in order to follow @ijuma suggestion? I have added the link to KAFKA-16384